flink-runtime/src/test/java/org/apache/flink/runtime/metrics/DescriptiveStatisticsHistogramStatisticsTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.metrics;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests for {@link DescriptiveStatisticsHistogramStatistics} and {@link
+ * DescriptiveStatisticsHistogramStatistics.CommonMetricsSnapshot}.
+ */
+class DescriptiveStatisticsHistogramStatisticsTest {
+
+    /** Tests the getPercentile functionality of the DescriptiveStatisticsHistogramStatistics. */
```

Comment:
The intention of this test is to make sure the instance percentilesImpl does not have null data.